extensions.json

```
@@ -518,6 +518,10 @@
       "version": "0.6.6",
       "checkout": "v0.6.6"
     },
+    {
+      "id": "lextudio.restructuredtext",
+      "repository": "https://github.com/vscode-restructuredtext/vscode-restructuredtext"
```

This project seems to maintain dedicated release tags, so I suggest we use them instead of publishing straight from the default branch.

```suggestion
      "repository": "https://github.com/vscode-restructuredtext/vscode-restructuredtext",
      "version": "129.0.0",
      "checkout": "129.0.0"
```